pkg/cmd/resources.go

```diff
@@ -2,17 +2,45 @@ package cmd
 
 import (
 	"fmt"
+	"io/ioutil"
+	"net/http"
 
+	"github.com/pelletier/go-toml"
```

Comment:
did we switch toml packages recently? We normally use `github.com/BurntSushi/toml` IIRC

Reply:
i will switch it back